modules/local/alevinqc.nf

```diff
@@ -1,23 +1,25 @@
 process ALEVINQC {
 
     //
-    // This module executes alevinfry QC reporting tool on alevin results
+    // This module executes alevinfry QC reporting tool on alevin-fry results
     //
 
     tag "$meta.id"
     label 'process_low'
 
-    //The alevinqc 1.14.0 container is broken, missing some libraries - thus reverting this to previous 1.12.1 version
-    conda "bioconda::bioconductor-alevinqc=1.12.1"
+    conda "bioconda::bioconductor-alevinqc=1.18.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/bioconductor-alevinqc:1.12.1--r41h9f5acd7_0' :
-        'biocontainers/bioconductor-alevinqc:1.12.1--r41h9f5acd7_0' }"
+        'https://depot.galaxyproject.org/singularity/bioconductor-alevinqc:1.18.0--r43hf17093f_0' :
```

Comment:
was the tool update required to work with simpleaf?
Ideally I would have seen two PRs, one for the module change as in this PR, and one for the tool update.

Anyways, it is already here and no need to change, but, would be good to add this info in the `CHANGELOG.md`.

Reply:
Perfect